src/prefect/tasks/control_flow/conditional.py

```diff
@@ -147,11 +156,15 @@ def merge(*tasks: Task, flow=None) -> Task:
             one of them will contain a result and the others will all be skipped.
         - flow (Flow, optional): The flow to use, defaults to the current flow
             in context if no flow is specified
+        - mapped (bool, optional): Whether the results of these tasks should be mapped over
+                with the specified keyword arguments; defaults to `False`.
```

Comment:
I don't think this docstring makes sense with the context of `merge` (same for `ifelse` and `switch`). Perhaps something more along the lines of:

```
If true, the `merge` operation will be mapped over the arguments instead of applied directly. Defaults to `False`.
```